<launch>

<!-- MoveBaseSeq node launch and Points Sequence ROS Parameters Loading -->

    <!--Run move_base_seqs node-->
        <node pkg="tb3_swarm" type="move_base_seq.py" name="move_base_seq" output="screen">
        <!-- Load Sequence of Points (Cartesian Coordinates wrt "map" RF, in meters) -->
            <param name="to_do_task" value="Ground"/>
        <!-- Load Sequence of Desired Yaw Angles (no rotations around x and y axes) in degrees-->
            <rosparam param="yea_seq">[0,90,0]</rosparam>

        </node>

        <node pkg="tb3_swarm" type="odom_movement_detector.py" name="odom_distance" output="screen"/>

</launch>
